src/components/non-bitgo.js

```diff
@@ -117,7 +118,7 @@ class NonBitGoRecoveryForm extends Component {
         return obj;
       }, {});
 
-      if (this.state.coin === 'btc' && this.state.apiKey) {
+      if (this.state.coin === 'bsv' && this.state.apiKey) {
```

Comment:
Shouldn't it be both btc and bsv here?

Reply:
we no longer need it for btc, because we are using blockstream for btc now :)